src/main/kotlin/com/sourcegraph/cody/chat/NewChatAction.kt

```diff
@@ -0,0 +1,20 @@
+package com.sourcegraph.cody.chat
+
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.project.DumbAwareAction
+import com.sourcegraph.cody.CodyToolWindowContent
+
+class NewChatAction : DumbAwareAction() {
```

Comment:
1. chat message `what is ternary search?`
2. send & quickly `New chat`
3. the reply is generated anyway 😕

Reply:
Fixed!